### PR TITLE
Added Implementation of Logging

### PR DESCRIPTION
**Pull request by daiahnadine**

### CODE REVIEW

File: employee-info-mgmt-system/src/main/java/Main.java [hunk 1]
```diff
@@ -4,17 +4,18 @@
 
 import java.util.List;
 import java.util.Scanner;
+import java.util.logging.Logger;
 
 public class Main {
-
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
```
Comment (RoCS2024) [inline employee-info-mgmt-system/src/main/java/Main.java:10]: for final static identifiers we use all caps. examples: LOGGER, SOME_VARIABLE, etc. please change accordingly

File: employee-info-mgmt-system/src/main/java/Main.java [hunk 2]
```diff
@@ -4,17 +4,18 @@
 
 import java.util.List;
 import java.util.Scanner;
+import java.util.logging.Logger;
 
 public class Main {
```
Comment (RoCS2024) [inline employee-info-mgmt-system/src/main/java/Main.java:9]: we do not use logging in Main class. revert to println

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Review by RoCS2024 — COMMENTED** [review]
Also please follow correct format of PRs as discussed earlier

**Comment by daiahnadine** [comment]
These comments are noted po Sir. Thank you po.

**Pull request closed without merging**